src/main/java/seedu/address/model/person/Person.java

```diff
@@ -109,15 +109,16 @@ public boolean isSamePerson(Person otherPerson) {
     }
 
     /**
-     * Returns true if both persons belong to the same region.
+     * Returns true if both persons belong to the suitable regions.
      * {@code otherPerson} must not be null.
      *
      * @param otherPerson Person to be compared to.
-     * @return True if both persons belong to the same region and false otherwise.
+     * @return True if both persons belong to the same region or at least one person
+     *      has no specified available dates, false otherwise.
```

Comment:
```suggestion
     * @return True if both persons belong to the same region or at least one person
     *      has no specified region, false otherwise.
```